tests/director/tests/formation_api_test.go

```diff
@@ -1021,6 +1021,7 @@ func TestFormationNotifications(stdT *testing.T) {
 		require.Equal(t, providerFormationName, assignedFormation.Name)
 		//list assignments - expect 0
 		assertFormationAssignments(t, ctx, subscriptionConsumerAccountID, formation.ID, 0, nil)
+		assertFormationStatus(t, ctx, subscriptionConsumerAccountID, formation.ID, graphql.FormationStatus{Condition: graphql.FormationStatusConditionReady, Errors: nil})
```

Comment:
In all tests only two (`READY`, `ERROR`) of the possible three Status Conditions are asserted. I think it is worth adding a test if no such is present where we assert that the Status is `IN_PROGRESS`.